pkg/game/player.go

```diff
@@ -0,0 +1,60 @@
+package game
```

Comment:
You could move it to separate package.

Reply:
I think this would be overdoing it a bit - `Player` is very closely related with/dependent on the `Game` and `Board` classes. I can ([and did](https://github.com/Jackenmen/Carcassonne-Engine/commit/b20a01aa5b91fb1132ae6b45cef28fa5a2fb3d67)) split it out but I don't think it's actually better than the alternative (and it did mean having to switch `player` tests from tests that can access the package's internals to one that can't - white box -> black box testing, I guess).

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_consider_fewer_larger_packages